oaknational/Oak-Web-Application
# Feat/subject filters

## Description

- Adds subject filters to search
- Refactors search filters to be more generic

## Issue(s)

Fixes #1584 

## How to test

1. Go to search page - click on subject filters
2. Click on _______
3. You should see _______

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist
